bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/HueBindingConstants.java

```diff
@@ -101,7 +101,9 @@ public class HueBindingConstants {
 
     public static final String NORMALIZE_ID_REGEX = "[^a-zA-Z0-9_]";
 
-    //
+    public static final String DISCOVERY_LABEL_PATTERN = "Philips Hue (%s)";
```

Comment:
It's interesting that they announce it as "Philips Hue" and not "Philips hue"...
Should we maybe nonetheless change the discovery label to "Philips hue" for all discovery services?

Reply:
They actually do announce it as "Philips hue". This was the reason for #14871. The label defined here is the user-visible thing label.

Reply:
Ah, ok. So we change it to "Hue", since we capitalize our labels, right?

Reply:
Do you mean that the correct spelling is actually "Philips hue"? I thought "hue" was only the branding they use graphically. 

Asked Chat GPT: _The correct spelling and casing for Philips Hue is with "Philips" capitalized and "Hue" lowercase, like this: "Philips Hue"._

😆

Reply:
> So we change it to "Hue", since we capitalize our labels, right?

I would say yes, but it is kind of conflicting, if "hue" is actually the correct spelling. TBH, not sure what would be best. Some companies use all UPPERCASE product names, which we probably usually change - for example NETATMO.

Reply:
Well, as you mentioned yourself, the textual announcement via mDNS is also "hue" - so yes, this seems to be the correct spelling according to Philips (and I think we used this throughout the binding as well).

Reply:
Let's then simply say, we use capitalization in our labels - then we are always right with "Hue" here. And that's what we did as a discovery result for 10 years now, so let's stick to it. 👍